src/components/user/EditForm.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { func, string, bool, object } from 'prop-types';
+import { Field, reduxForm } from 'redux-form/immutable';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import {
+  injectIntl,
+  intlShape,
+  defineMessages,
+  FormattedMessage
+} from 'react-intl';
+
+import Routes from '../../constants/routesPaths';
+import Loading from '../common/Loading';
+import Input from '../common/Input';
+import SelectBox from '../common/Select';
+import { validations, edit } from '../../utils/constraints';
+import smilies from '../../img/smilies.svg';
+import close from '../../img/close.png';
+import profilePic from '../../img/guy.svg';
+import blueCircle from '../../img/blue-circle.svg';
+
+const messages = defineMessages({
+  name: { id: 'signup.name' },
+  firstName: { id: 'edit.firstName' },
+  lastName: { id: 'edit.lastName' },
+  email: { id: 'login.form.email' },
+  gender: { id: 'signup.gender' },
+  genderSelect: { id: 'signup.genderSelect' },
+  success: { id: 'edit.success' }
+});
+
+const options = [
+  { value: 'male', label: 'Male' },
+  { value: 'female', label: 'Female' },
+  { value: 'other', label: 'Other' }
+];
```

Comment:
Wouldn't you have to use the localization here too? What would happen if for example you were using the app in Spanish and then the options of the select list show up? I know you are not updating the Spanish locales but it would be good to be consistent and use the localization here too.